integration-tests/start.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+type tokenBody struct {
+	Token string
+}
+
+func main() {
+
+	// Make a user POST request
+	msg := []byte(`{"email":"test@test.com", "password":"abcdef"}`)
+	postRs, postErr := http.Post("http://localhost:8180/users", "application/json", bytes.NewBuffer(msg))
```

Comment:
I think a good improvement could be to receive the fog address via arguments. What do you think?
Take a look [here](https://gobyexample.com/command-line-arguments).

Reply:
I modified the code. It should create and return the token. For now, you use `go build start.go` and execute using
`./start "" "" "" ""`

Reply:
As we discussed in the slack, it'd be good those empty strings not to be required.

Reply:
Done. If you want to put arguments you must use `go build start.go` and `./start <email> <password> <server> <port>`.